tests/test_phantomjs_manager.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import pytest
+from selenium import webdriver
+
+from tests.test_cache import cache, delete_cache
+from webdriver_manager.phantomjs import PhantomJsDriverManager
+
+PATH = '.'
+
+
+def delete_old_install(path=None):
+    if path is None:
+        delete_cache()
+    else:
+        path = os.path.abspath(path)
+        for f in os.listdir(path):
+            if f.startswith("phantomjs-"):
+                try:
+                    os.remove(os.path.join(path, f))
+                except:
+                    pass
+
+
+
+def test_phantomjs_manager_with_specific_version():
+    bin = PhantomJsDriverManager("2.0.0").install()
```

Comment:
`bin = PhantomJsDriverManager("2.0.0", os_type="win32").install()`
would be appropriate for this test to be working on any OS